pkgs/tools/compression/zstd/default.nix

```diff
@@ -49,12 +49,14 @@ stdenv.mkDerivation rec {
   preInstall = stdenv.lib.optionalString enableShared ''
     substituteInPlace ../programs/zstdgrep \
       --replace ":-grep" ":-${gnugrep}/bin/grep" \
-      --replace ":-zstdcat" ":-$out/bin/zstdcat"
+      --replace ":-zstdcat" ":-$bin/bin/zstdcat"
 
     substituteInPlace ../programs/zstdless \
-      --replace "zstdcat" "$out/bin/zstdcat"
+      --replace "zstdcat" "$bin/bin/zstdcat"
   '';
 
+  outputs = [ "bin" "lib" "dev" "man" "out" ];
+
```

Comment:
This makes `.out` completely empty.  I believe in this case we still just omit `"lib"` so that libraries remain in the main output ([some docs](https://nixos.org/nixpkgs/manual/#sec-multiple-outputs-) but not saying explicitly).